{"name": "test_loginFailMethod", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <AppiumFramwork.tests.LoginTest.LoginTest testMethod=test_loginFailMethod>\n\n    @pytest.mark.run(order=3)\n    def test_loginFailMethod(self):\n        cl.allureLogs(\"App Opened\")\n        self.gt.clickLoginBotton()\n        self.gt.enterEmail()\n        self.gt.enterPassword2()\n        self.gt.clickOnLoginB()\n>       self.gt.verifyAdminScreen()\n\nLoginTest.py:37: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <AppiumFramwork.pages.LoginPage.LoginPageTest object at 0x000001D6C30A0400>\n\n    def verifyAdminScreen(self):\n        adminScreen = self.isDisplayed(self._pageTitle,\"text\")\n>       assert adminScreen == True\nE       AssertionError\n\n..\\pages\\LoginPage.py:44: AssertionError"}, "steps": [{"name": "App Opened", "status": "passed", "start": 1653572352598, "stop": 1653572352598}, {"name": "Click on Login Button", "status": "passed", "start": 1653572353316, "stop": 1653572353316}, {"name": "Entered email id", "status": "passed", "start": 1653572354928, "stop": 1653572354929}, {"name": "Entered Password", "status": "passed", "start": 1653572356079, "stop": 1653572356079}, {"name": "Clicked on Login Button in Login Screen", "status": "passed", "start": 1653572356638, "stop": 1653572356638}], "attachments": [{"name": "text", "source": "edf93719-dd3b-4ec8-9f28-37c56798ca7c-attachment.png", "type": "image/png"}, {"name": "text", "source": "b9fab820-676e-4eb6-928a-4831117392ce-attachment.png", "type": "image/png"}, {"name": "log", "source": "eed8563e-13ae-4d1b-96ba-4da364a7a368-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "932cf512-e530-44fc-88a8-b2a3a8169b4f-attachment.txt", "type": "text/plain"}], "start": 1653572352598, "stop": 1653572383537, "uuid": "095760af-6339-441a-b6b1-1322115f2e69", "historyId": "aeb6c84858cb897d59574054d46d09e3", "testCaseId": "1512c76b4b2d60f059541c0f73c45353", "fullName": "TestSuite.LoginTest#test_loginFailMethod", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('beforeClass', 'beforeMethod')"}, {"name": "tag", "value": "run(order=3)"}, {"name": "suite", "value": "TestSuite"}, {"name": "subSuite", "value": "LoginTest"}, {"name": "host", "value": "BTZVBK3"}, {"name": "thread", "value": "5660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestSuite"}]}